documentation/dev-portal/src/nymvpn/gui-linux.md

```diff
@@ -33,16 +34,16 @@ sudo apt-get install -f ./nym-vpn_0.0.2_amd64.deb
 
 ### Configuration
 
-7. Create a NymVPN config directory called `nym-vpn` in your `~/.config`, either manually or by a command:
+6. Create a NymVPN config directory called `nym-vpn` in your `~/.config`, either manually or by a command:
 ```sh
 mkdir $HOME/.config/nym-vpn/
```

Comment:
shouldn't this be `mkdir -p`?

Reply:
doesn't this asume that you may not have `~/.config` at all? We have it without `-p` in the scripts as well and so far no complains, your call .)